src/stories/components/Badge.stories.mdx

```diff
@@ -67,9 +67,10 @@ export const docsDecorator = () => {
 <br />
 
 ## Recomendamos o uso de Badges quando:
-- Você quiser mostrar status associados com a lógica de negócio ou açoes do usuário.
-- O conteúdo a ser mostrado for *readonly*
-- For preciso categorizar algo
+- For necessário mostrar status associados com a lógica de negócio ou ações do usuário.
+- O conteúdo a ser mostrado for readonly.
```

Comment:
O itálico na versão anterior era pra indicar o estrangeirismo. Tu acha que a gente precisa tirar?

Reply:
Eu realmente tinha esquecido de colocar itálico depois de ajustar o texto.